app/lib/team.js

```diff
@@ -260,7 +281,7 @@ async function isModerator (teamId, osmId) {
   if (!osmId) throw new Error('osm id is required as second argument')
   const conn = await db()
   const [{ count }] = await conn('moderator').where({ team_id: teamId, osm_id: osmId }).count()
```

Comment:
I think we should stick to one style, this doesn't have `unpack` and maybe should use `length` like the previous comments. What do you think @guidorice

Reply:
I like your suggestions to make the code simpler/shorter. Will make this change.